www/base/src/components/MultipleWorkersActionsModal/MultipleWorkersActionsModal.tsx

```diff
@@ -36,6 +37,42 @@ const workerToSelectOption = (worker: Worker): SelectOption => {
   return { value: worker, label: worker.name }
 }
 
+const Control = (props: any) => {
+  const { useRegexSearch, handleToggleClick, selectRef } = props;
```

Comment:
Better not use `use` as a prefix for variables, because React convention is to `use` for hooks. https://legacy.reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook